editor/editor_help_search.cpp

```diff
@@ -510,6 +490,28 @@ bool EditorHelpSearch::Runner::_phase_select_match() {
 	return true;
 }
 
+void EditorHelpSearch::Runner::_match_method_name_and_push_back(Vector<DocData::MethodDoc> &p_methods, Vector<DocData::MethodDoc *> *p_match_methods) {
```

Comment:
```suggestion
void EditorHelpSearch::Runner::_match_method_name_and_push_back(Vector<DocData::MethodDoc> &p_methods, Vector<DocData::MethodDoc *> *r_match_methods) {
```
Should use `r_` prefix if it's modified (used as a "return" value).

Reply:
Done